Generator/source/CMakeLists.txt

```diff
@@ -74,42 +120,42 @@ set(SOURCES
   generators/Genetic/GenGenerator.cpp
   DataBase/DataBaseGenerator.cpp
   DataBase/DataBaseGeneratorParameters.cpp
+  generatorAPI.cpp
```

Comment:
What about using lexicographic order for all the file-related arguments of `set`? Probably, except `main.cpp`, but I'm not sure about popular conventions here.